widgets/price-table/tpl/atom.php

```diff
@@ -24,15 +25,15 @@
 				<div class="ow-pt-per"><?php echo esc_html($column['per']) ?></div>
 			</div>
 
-			<?php if(!empty($column['image'])) : ?>
-				<div class="ow-pt-image">
-					<?php $this->column_image($column) ?>
-				</div>
-			<?php endif; ?>
+            <div class="ow-pt-image">
```

Comment:
What's the main reason for switching this around? Wouldn't this result in empty `ow-pt-image` divs if there's no row image set? That feels a little messy.

There are also a few lines below this one that use space instead of tab indenting.